src/main/groovy/org/mockito/release/internal/gradle/util/GitUtil.java

```diff
@@ -46,4 +46,11 @@ public static Object getGitGenericUserNotation(ReleaseConfiguration conf) {
     public static String getTag(ReleaseConfiguration conf, Project project) {
         return conf.getGit().getTagPrefix() + project.getVersion();
     }
+
+    /**
+     * Returns Git commit message based on release configuration and the given message
+     */
+    public static String getCommitMessage(ReleaseConfiguration conf, String message) {
```

Comment:
Good idea to move this method here and add unit test. This makes the code easier to test and the solution is cleaner, too.